src/plugins/telemetry_collection_manager/server/types.ts

```diff
@@ -82,7 +82,9 @@ export interface BasicStatsPayload {
   version: string;
   cluster_stats: object;
   collection?: string;
-  stack_stats: object;
+  // Needed for testing to make fields easier to access
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  stack_stats: Record<string, any>;
```

Comment:
`any` is a footgun, if we want to use it to avoid real types I think we should rather cast to any in the test. Even if it means we're having to repeat it a bit more.

Reply:
I see your point! I reverted this change and specified this test-friendlier type in the FTR helper in [`4c813a5` (#151808)](https://github.com/elastic/kibana/pull/151808/commits/4c813a525ebabb6300ef65c3543c92619169f163)